docs/technology/glossary.md

```diff
@@ -0,0 +1,49 @@
+---
+id: glossary
+title: Glossary
+---
+
+**Metachain**: the blockchain that runs in a special shard, where the main responsibilities are not
```

Comment:
do the newlines look ok on the actual docs.elrond.com?

Reply:
yes, they look fine

Reply:
ok